index.js

```diff
@@ -1,11 +1,30 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.lenght = this.items.length;
+  }
+  
+  add(item) {
+    this.items.push(item);
+    this.lenght = this.items.length;
+    this.items.sort((a, b) => a - b);
+  }
 
-  add(item) {}
+  get(pos) {
+    if (pos > this.items.length) {
+      throw new Error ('OutOfBounds');
+    } else {
+      return this.items(pos);
+    }
+  }
 
-  get(pos) {}
-
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error ('EmptySortedList');
+    } else {
+      return Math.max(...this.items);
+    }
+  }
 
   min() {}
```

Comment:
en este de min puedes usar la misma estructura de max pero lo contrario min

` if (this.length === 0) {
      throw new Error ('EmptySortedList');
    } else {
      return Math.min(...this.items);
    }`

Reply:
te dejo las otras dos funciones 

`
  sum() {
    if (this.length === 0 ){
      return 0
    }else {
      return this.items.reduce((a, b) => a + b, 0)
    }
  }

  avg() {

    if(this.length === 0){
      throw new Error('EmpySortedList');
    } else {
      return this.sum()/this.length;
    }
  }`